README.md

```diff
@@ -39,3 +39,67 @@ linuxkit build -docker -format iso-bios linux.yml
 
 (if you get a VFS error booting this image, you may need to try other formats like `raw-bios` or `raw-efi`)
 ```
+
+## Unittests
+
+Unittests are run in a docker container, that is build before the tests, so they can be run on Linux or Mac. (I have not tried 
+Windows). Coverage files can be found after run in out/coverage.html.
+
+```bash
+make unittest
+```
+
+One test can be run with following make target:
+
+```bash
+make unittest-one TEST_PATTERN=<test name>
+```
+
+## Local verification setup
+
+Disclaimer: This setup will probably not work for you, but it can remind me in a few months how to set this thing up.
+Used virtualization `libvirt + KVM`.
+
+### Prepare binary and served files
+
+1. Follow steps from build section, about assets and talos configs. 
+2. Set required capabilities on the talos-pxe binary:
+    ```bash
+    sudo setcap cap_net_raw,cap_net_admin,cap_net_bind_service+eip ./talos-pxe
+    ```
+
+### Setup for verifying DHCP no proxy mode
```

Comment:
Don't forget additional firewall rules that are needed on RHEL/Fedora:
```
sudo firewall-cmd --add-port=4011/tcp --zone=libvirt
sudo firewall-cmd --add-interface=eth0 --zone=libvirt
sudo firewall-cmd --add-interface=eth1 --zone=libvirt
````

Reply:
I also have to do:
```
sudo ip route del default via 192.168.122.1 dev eth0
```

since talos-pxe programs the GW.

Reply:
at which point you need to do:
```sudo ip route del default via 192.168.122.1 dev eth0```

also this looks like it is for proxy setup or is it for both ?

```sudo firewall-cmd --add-port=4011/tcp --zone=libvirt
sudo firewall-cmd --add-interface=eth0 --zone=libvirt
sudo firewall-cmd --add-interface=eth1 --zone=libvirt```

Reply:
> at which point you need to do:
sudo ip route del default via 192.168.122.1 dev eth0

After talos-pxe has configured itself (as it sets the default route, overriding whatever the computer had in place).

> also this looks like it is for proxy setup or is it for both ?

Correct, as it's the one requiring the veth pair.